pyanaconda/timezone.py

```diff
@@ -136,15 +148,14 @@ def get_all_regions_and_timezones():
 
     result = OrderedDict()
 
-    for tz in pytz.common_timezones:
+    for tz in sorted(all_timezones()):
```

Comment:
It might be better to move the sorting into function itself to take advantage of the cache.

Reply:
Technically, it could. However the other function that uses this does not need sorting and it appeared to me that `get_all_regions_and_timezones()` is usually called just once? If it is called multiple times, should that be cached as well?